core/src/main/protobuf/schema.proto

```diff
@@ -15,9 +15,10 @@ message Type {
 message Declaration {
     required string name = 1;
     required string kind = 2;
-    required bool isImplicit = 3;
```

Comment:
Won't this change the binary representation?

Reply:
Yes it did, and the version in the servers is up to date.